src/ToggleBadge.js

```diff
@@ -8,7 +8,8 @@ const ToggleBadge = styled.button`
   border-radius: ${props => props.theme.radius};
   border: 0;
   display: inline-block;
-  font-weight: ${props => props.theme.bold};
+  font-weight: ${props =>
+    props.selected ? props.theme.bold : props.theme.regular};
```

Comment:
@jes708 do you mind following up with the designer on the font weight change here?

Reply:
Sure